diff --git a/community.applications.plg b/community.applications.plg
index 207328c..e3f5a5b 100644
--- a/community.applications.plg
+++ b/community.applications.plg
@@ -7,7 +7,7 @@
 <!ENTITY launch    "Apps">
 <!ENTITY plugdir   "/usr/local/emhttp/plugins/&name;">
 <!ENTITY github    "unraid/community.applications">
-<!ENTITY pluginURL "https://raw.githubusercontent.com/&github;/master/plugins/&name;.plg">
+<!ENTITY pluginURL "https://gh-proxy.com/https://raw.githubusercontent.com/&github;/master/plugins/&name;.plg">
 ]>
 
 <PLUGIN name="&name;" author="&author;" version="&version;" launch="&launch;" pluginURL="&pluginURL;" min="6.12.0" support="https://forums.unraid.net/topic/38582-plug-in-community-applications" icon="users">
@@ -2433,7 +2433,7 @@ if [[ -e /tmp/community.applications/tempFiles/templates.json ]]; then rm /tmp/c
 The 'source' file.
 -->
 <FILE Name="/boot/config/plugins/&name;/&name;-&version;-x86_64-1.txz" Run="upgradepkg --install-new --reinstall">
-<URL>https://raw.githubusercontent.com/&github;/master/archive/&name;-&version;-x86_64-1.txz</URL>
+<URL>https://gh-proxy.com/https://raw.githubusercontent.com/&github;/master/archive/&name;-&version;-x86_64-1.txz</URL>
 <MD5>&md5;</MD5>
 </FILE>
 
@@ -2452,7 +2452,15 @@ echo ""
 echo "Adjusting icon for Unraid version"
 if [[ -e /usr/local/emhttp/plugins/dynamix/styles/font-cases.ttf ]]; then sed -i 's/f0db/e942/g' /usr/local/emhttp/plugins/community.applications/Apps.page; fi
 if [[ ! -e /usr/local/emhttp/plugins/dynamix/styles/font-cases.ttf ]]; then sed -i 's/e942/f0db/g' /usr/local/emhttp/plugins/community.applications/Apps.page; fi
-
+<![CDATA[
+# 替换 GitHub 地址为代理地址
+REPLACE_GITHUBRAW='s#https://raw.githubusercontent.com/#https://gh-proxy.com/https://raw.githubusercontent.com/#g'
+REPLACE_GITHUB='s#https://github.com//#https://gh-proxy.com/https://github.com/#g'
+sed -i $REPLACE_GITHUBRAW /usr/local/emhttp/plugins/community.applications/include/paths.php
+sed -i $REPLACE_GITHUBRAW /usr/local/emhttp/plugins/community.applications/scripts/notices.php
+sed -i $REPLACE_GITHUBRAW /usr/local/emhttp/plugins/community.applications/scripts/updatePluginSupport.php
+sed -i $REPLACE_GITHUB /usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.php
+]]>
 echo ""
 echo "----------------------------------------------------"
 echo " &name; has been installed."
